src/DynamoCore/Core/DynamoMigrator.cs

```diff
@@ -401,4 +424,36 @@ protected override DynamoMigratorBase MigrateFrom(DynamoMigratorBase sourceMigra
             return base.MigrateFrom(sourceMigrator);
         }
     }
+
+    class MigratorPathResolver : IPathResolver
+    {
+        private readonly IEnumerable<string> emptyList = new List<string>();
+        public MigratorPathResolver(string userDataRoot)
+        {
+            UserDataRootFolder = userDataRoot;
+        }
+
+        public IEnumerable<string> AdditionalResolutionPaths
+        {
+            get { return emptyList; }
```

Comment:
Is this a temporary code? If not perhaps we can do `return Enumerable.Empty<string>();` instead? Just a minor thing though.